docs/01-getting-started/01-installation.mdx

```diff
@@ -69,9 +69,15 @@ These scripts refer to the different stages of developing an application:
 - `start`: runs [`next start`](/docs/app/api-reference/next-cli#production) to start a Next.js production server.
 - `lint`: runs [`next lint`](/docs/app/api-reference/next-cli#lint) to set up Next.js' built-in ESLint configuration.
 
-### Create the `app` folder
+### Creating directories
 
-Next, create an `app` folder and add a `layout.tsx` and `page.tsx` file. These will be rendered when the user visits the root of your application.
+Next.js uses file-system routing, which means how you structure your files determines the routes in your application.
+
+#### The `app` folder
+
+For new applications, we recommend using the `app` router. This router allows you to use React's latest features, and is currently where we're focusing our development efforts.
```

Comment:
```suggestion
For new applications, we recommend using the `app` router. This router allows you to use React's latest features and is an evolution of the `pages` router based on community feedback.
```

Reply:
Should we use "App Router" and "Pages Router" instead?